### PR TITLE
fix: gitea initcontainer ca volumeMount name fixed

### PR DESCRIPTION
**Pull request by mojtabaimani**

### CODE REVIEW

**Review by Dunky13 — APPROVED** [review]
Nasty little bugger, lgtm

**Review by Morriz — COMMENTED** [review]
The setup is working in multiple locations successfully (I can recall Drone as well). If it only fails for gitea I start to wonder if this is the right fix

**Comment by Dunky13** [comment]
It was mounting a wrong volume. Not sure how it could've worked in previous runs? Maybe not spotted due to production? But it tried to mount a secret, not the certs. This change actually mounts the certs

**Comment by Morriz** [comment]
Did something change in the setup since then?

**Comment by mojtabaimani** [comment]
when you add docker imagePullSecret directly in the pod they will not take the gcr imagePullSecret form the default service account anymore. I added the gcr imagePullSecret to the pods as well.

File: values/drone/drone-admit-members.gotmpl [hunk 1]
```diff
@@ -29,4 +29,5 @@ podAnnotations:
 image:  
   pullSecrets:
     - name: otomi-pullsecret-global
+    - name: gcr-json-key
```
Comment (Morriz) [inline values/drone/drone-admit-members.gotmpl:32]: Why are you adding a pullsecret here for otomi api?

File: values/drone/drone.gotmpl [hunk 2]
```diff
@@ -112,4 +112,5 @@ extraRootCA: |
 {{- with .Values.otomi | get "globalPullSecret" nil }}
 imagePullSecrets:
   - name: otomi-pullsecret-global
+  - name: gcr-json-key
```
Comment (Morriz) [inline values/drone/drone.gotmpl:115]: Same? Are you just trying things and not confirming the effects?

**Review by Morriz — CHANGES_REQUESTED** [review]
This looks like another PR with changes that make no sense to me, and I have no clue what you did to come to this change, or why and what it would solve.
I would like to ask you to write out what you intend to do, what you decided, how you think to solve something, and how you tested your work, and wether you can conclude it solves the problem at hand.

Sorry to be so anal but we can't afford devs to suggest changes that introduce instability or bad code

File: values/drone/drone-admit-members.gotmpl (hunk 1 above)
Comment (svatwork) [reply values/drone/drone-admit-members.gotmpl:32]: I had to add this Secret in `otomi-eks-dev` or else it threw a pull error, it doesn't make sense to you? I am curious what the alternative solution is (also for my learning experience)?

Comment (svatwork) [reply values/drone/drone-admit-members.gotmpl:32]: I didn't understand @Morriz comment, but he means like "why is it added in this namespace", looking at it now I wonder if it is relevant in this context? Did you verify it @mojtabaimani?

**Review by svatwork — CHANGES_REQUESTED** [review]

**Review by Morriz — CHANGES_REQUESTED** [review]

**Review by Morriz — APPROVED** [review]

**Pull request merged**